endpoints/getting-started/src/main/appengine/openapi.yaml

```diff
@@ -3,7 +3,7 @@ info:
   description: "A simple Google Cloud Endpoints API example."
   title: "Endpoints Example"
   version: "1.0.0"
-host: "YOUR-PROJECT-ID.appspot.com"
+host: "echo-api.endpoints.YOUR-PROJECT-ID.cloud.goog"
```

Comment:
This field is not a DNS name that the application will be called at (not yet at least). It is the Endpoints service name.